shared_model/validators/protobuf/proto_transaction_validator.hpp

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_PROTO_TRANSACTION_VALIDATOR_HPP
+#define IROHA_PROTO_TRANSACTION_VALIDATOR_HPP
+
+#include "backend/protobuf/transaction.hpp"
+#include "validators/transaction_validator.hpp"
+
+namespace shared_model {
+  namespace validation {
+
+    template <typename FieldValidator, typename CommandValidator>
+    class ProtoTransactionValidator
+        : public TransactionValidator<FieldValidator, CommandValidator> {
+     private:
+      Answer validateProtoTx(
+          const iroha::protocol::Transaction &transaction) const {
+        Answer answer;
+        std::string tx_reason_name = "Protobuf Transaction";
+        ReasonsGroupType reason(tx_reason_name, GroupedReasons());
+        for (const auto &command :
+             transaction.payload().reduced_payload().commands()) {
+          switch (command.command_case()) {
+            case iroha::protocol::Command::COMMAND_NOT_SET: {
```

Comment:
Code blocks in cases are redundant, please consider removing.

Reply:
Nevertheless google style doesn't forbid this way, I agree that variant with braces looks uglier and harder to read